sources/iTermNotificationController.m

```diff
@@ -84,6 +85,53 @@ - (BOOL)notify:(NSString *)title
                       sticky:NO];
 }
 
+- (BOOL)notifyRich:(NSString *)title
+      withSubtitle:(NSString *)subtitle
+   withDescription:(NSString *)description
+         withImage:(NSString *)image
+         withSound:(NSString *)sound
+       windowIndex:(int)windowIndex
+          tabIndex:(int)tabIndex
+         viewIndex:(int)viewIndex {
+    DLog(@"Notify title=%@ description=%@ window=%@ tab=%@ view=%@",
+            title, description, @(windowIndex), @(tabIndex), @(viewIndex));
+
+    if (description == nil) {
+        DLog(@"notifyRich passed nil description.");
+        return NO;
+    }
+
+    // Send to NSUserDefaults directly
+    NSUserNotification *notification = [[NSUserNotification alloc] init];
+    notification.informativeText = description;
+
+    NSDictionary *context = nil;
+    if (windowIndex >= 0) {
+        context = @{ @"win": @(windowIndex),
+                @"tab": @(tabIndex),
```

Comment:
Inconsistent indentation. I suggest moving `@"win":…` onto the next line and then xcode will do the right thing.

Reply:
Fixed.